ContentPatcher/docs/i18n/zh/author-guide/action-editimage.md

```diff
@@ -0,0 +1,107 @@
+﻿← [模组作者指南](../author-guide.md)
+
+一个含有 **`"Action": "EditImage"`** 的补丁会更改游戏已加载的图像的一部分。任意数量的内容包都可以编辑同一素材。你可以用补丁向下延伸图像（Content Patcher将扩展图像以适应新图像）。
+
+**🌐 其他语言： [en (English)](../../../author-guide/action-editimage.md)。**
+
+## 目录<a name="contents"></a>
+* [用法](#usage)
+  * [格式](#format)
+  * [示例](#examples)
+* [参见](#see-also)
+
+## 使用<a name="usage"></a>
+### 格式<a name="format"></a>
+
+一个`EditImage`补丁是`Changes`下含有此字段的模型 ([下文有例子](#examples))
+
+<dl>
+<dt>必填字段：</dt>
+<dd>
+
+字段       | 用途
+--------- | -------
+`Action`  | 要进行的更改类型。此操作类型设置为`EditImage`。
+`Target`  | 需编辑的[游戏素材名](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），比如`Portraits/Abigail`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+`FromFile` | 内容包文件夹中要修补到目标中的图像的相对路径（例如`assets/dinosaur.png`），或多个逗号分隔的路径。这可以是`.png`或`.xnb`文件。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+
+</dd>
+<dt>可选字段：</dt>
+<dd>
+
+字段         | 用途
+----------- | -------
+`FromArea`  | <p>源图片中需拷贝到目标的部分，默认整个源图片</p><p>此字段是一个对象，含有左上角点的X和Y像素坐标，和区域的像素大小，长（Width）与高（Height）。该对象的字段支持[令牌](../author-guide.md#tokens)。</p>
+`ToArea`    | <p>目标图片中要替换的部分。默认大小与 `FromArea` 相同，位于贴图的左上角。</p><p>此字段是一个含有左上角点的X和Y像素坐标区域的长（Width）与高（Height）的对象。该对象的字段支持[令牌](../author-guide.md#tokens)。</p><p>如果你指定的区域超出了图像的底部，Content Patcher将自动调整图像大小以适应新图像。</p>
+`PatchMode` | <p>如何将 `FromArea` 应用于 `ToArea`。默认为 `Replace`。</p> 可使用的值: <ul><li><code>Replace</code>: 用源图像替换目标区域中的每个像素。如果源图像有透明像素，则目标图像将在那里变为透明。</li><li><code>Overlay</code>: 在目标区域上绘制源图像。如果源图像有透明或半透明像素，则目标图像将“显示”这些像素。不透明像素将替换目标像素。</li></ul>例如，假设你的源图像是具有透明背景的河豚，而目标图像是实心绿色正方形。 以下是它们在不同`PatchMode`下的组合：<br />![](../../../screenshots/patch-mode-examples.png)
```

Comment:
The description for the `Overlay` patch mode might be incorrect:
> **en:** The red tile replaces the ground on the `Back` layer, but the ground is visible under the blue `Buildings` tile.
> **zh:** `Back`图层的红图块代替了`Back`图层的地面，而`Buildings`图层的蓝图块没有替换任何图块，地面依旧可见。

The translation seems to say that the `Buildings` layer doesn't replace any tiles; if so that's incorrect, since it replaces the tile on the `Buildings` layer (just not the `Back` ground tile under it).